js/pulse.js

```diff
@@ -806,7 +806,8 @@
          */
         this.sharedElementGive = function() {
             setTimeout(function(){
-                _adManager.ui.ooyalaVideoElement[0].play();
+                _adManager.sharedVideoElement.style.display = "block";
+                _adManager.sharedVideoElement.play();
             }, 100);
             _adManager.sharedVideoElement.style.visibility ="hidden";
```

Comment:
Correct, we shouldn't need to edit the visibility.  Have you tried debugging in to the video controller and seeing if there is a bug there?  If this plugin edits the visibility and the VTC does, then we can get out of sync. Which is what I suspect may be the root cause.